azurerm/internal/services/containers/kubernetes_nodepool.go

```diff
@@ -133,6 +133,11 @@ func SchemaDefaultNodePool() *schema.Schema {
 					Computed:     true,
 					ValidateFunc: validation.StringIsNotEmpty,
 				},
+				"proximity_placement_group_id": {
+					Type:         schema.TypeString,
+					Optional:     true,
```

Comment:
the VMSS resources (which this is provisioning under the hood) requires this to be ForceNew, so I assume this does too?

```suggestion
					Optional:     true,
					ForceNew: true,
```

Reply:
Ah, wasn't sure, docs/API wasn't stating this clearly. Let's make it so.